frontend/src/components/chat-input.tsx

```diff
@@ -51,6 +51,8 @@ export function ChatInput({
       if (files.length > 0) {
         event.preventDefault();
         onImagePaste(files);
+        // Clear the text input when an image is pasted
+        onChange?.("");
```

Comment:
```suggestion
```

Avoid directly calling this function like this here. `ChatInput` is meant to allow the developer to define the behaviour through the `onChange` prop. In this case, the component that uses `ChatInput` (e.g., `InteractiveChatBox`) should define this behaviour.